corehq/apps/users/management/commands/grant_web_apps_to_default_mobile_worker_role.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.analytics import get_role_user_count
+from corehq.apps.users.models_role import Permission, UserRole
+
+
+class Command(BaseCommand):
+    help = "Add access_web_apps permission to default mobile worker roles"
+
+    def handle(self, **options):
+        permission, created = Permission.objects.get_or_create(value='access_web_apps')
+        roles_to_update = UserRole.objects.filter(is_commcare_user_default=True)
+        for role in roles_to_update:
+            permissions = role.permissions
+            if not permissions.access_web_apps:
+                count = get_role_user_count(role.domain, role.couch_id, web_users_only=True)
```

Comment:
I'm expecting this call to make this script potentially quite slow on the first run. On subsequent runs, most/all roles should have had `access_web_apps` turned on already, so this call will only be hit for any remaining roles (and I can query before merging the migration to see how many that is).